plugins/modules/azure_rm_aduser.py

```diff
@@ -70,11 +70,24 @@
             - The mail alias for the user.
             - Used when either creating or updating a user account.
         type: str
-    password_profile:
+    password:
         description:
             - The password for the user.
             - Used when either creating or updating a user account.
         type: str
+        aliases:
+            - password_profile
+    password_force_change:
+        description:
+            - Whether or not the user will be forced to change their password at next logon.
+            - Note: If unspecified, Azure defaults this to true for new users.
```

Comment:
DOCUMENTATION.options.password_force_change.description.  Must be a string 
```suggestion
            - If unspecified, Azure defaults this to true for new users.
```